src/components/LanguageSelector.tsx

```
@@ -3,25 +3,25 @@
+import { slugify } from "@/utils/slugify";
 
-import SubLanguageSelector from "./SubLanguageSelector";
+// import SubLanguageSelector from "./SubLanguageSelector";
+import { useRouter } from "next/router";
 
 const LanguageSelector = () => {
-  const navigate = useNavigate();
+  const router = useRouter();
 
-  const { language, subLanguage, setSearchText } = useAppContext();
+  const { selectedLanguage, selectedCategory, setSearchText } = useAppContext();
```

ESLint

[error] 'selectedLanguage' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)